### PR TITLE
Adds Node.js documentation for code level metrics

### PR DESCRIPTION
**Pull request by bizob2828**
## Give us some context

* What problems does this PR solve?
    * Adds configuration and page about Code Level Metrics for Node.js
    * It was modeled after the .NET #7949

### CODE REVIEW

**Review by jordigh — DISMISSED** [review]

File: src/content/docs/apm/agents/nodejs-agent/installation-configuration/codestream-integration.mdx [hunk 1]
```diff
@@ -0,0 +1,102 @@
+---
+title: New Relic CodeStream integration for Node.js
+tags:
+  - Agents
+  - Nodejs agent
+  - Configuration
+metaDescription: The Node.js agent can be integrated with New Relic CodeStream to provide code level metrics.
+---
+
+Display context-sensitive APM data directly in your IDE by integrating [New Relic CodeStream](/docs/codestream/how-use-codestream/performance-monitoring#code-level) with the Node.js agent. Visualize code-level production telemetry in your editor as your write and review code with this integration.
```
Comment (a-sassman) [inline src/content/docs/apm/agents/nodejs-agent/installation-configuration/codestream-integration.mdx:10]:
```suggestion
Display context-sensitive APM data directly in your IDE by integrating [New Relic CodeStream](/docs/codestream/how-use-codestream/performance-monitoring#code-level) with the Node.js agent. This integration visualizes code-level production telemetry in your editor as your write and review code.
```

File: src/content/docs/apm/agents/nodejs-agent/installation-configuration/codestream-integration.mdx [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+
+## Get started [#get-started]
+
+First, [install](/docs/codestream/start-here/install-codestream) the New Relic CodeStream extension into your supported IDE of choice, and log in.
```
Comment (a-sassman) [inline src/content/docs/apm/agents/nodejs-agent/installation-configuration/codestream-integration.mdx:14]: Just to confirm, they're logging in _before_ they install, not _after_?

File: src/content/docs/apm/agents/nodejs-agent/installation-configuration/codestream-integration.mdx [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+
+<Callout variant="important">
+  The New Relic CodeStream integration is available in [Node.js agent version 9.7.5 and higher](/docs/release-notes/agent-release-notes/net-release-notes) and is disabled by default. To change this configuration, check out our [documentation](/docs/apm/agents/nodejs-agent/installation-configuration/nodejs-agent-configuration#code_level_metrics).
+</Callout>
+
+## Agent attributes [#attributes]
+
+The Node.js agent reports and attaches the following attributes to most spans produced by [web frameworks](https://docs.newrelic.com/docs/apm/agents/nodejs-agent/getting-started/compatibility-requirements-nodejs-agent/#support) in your application.
```
Comment (a-sassman) [inline src/content/docs/apm/agents/nodejs-agent/installation-configuration/codestream-integration.mdx:22]:
```suggestion
The Node.js agent reports and attaches the following attributes to most spans produced by [web frameworks](/docs/apm/agents/nodejs-agent/getting-started/compatibility-requirements-nodejs-agent#support) in your application.
```

File: src/content/docs/apm/agents/nodejs-agent/installation-configuration/codestream-integration.mdx [hunk 4]
```diff
@@ -0,0 +1,102 @@
...
+
+<Callout variant="important">
+  In some cases, the agent will directly wrap base framework code that is invoked within your application and will not display within your IDE.
```
Comment (a-sassman) [inline src/content/docs/apm/agents/nodejs-agent/installation-configuration/codestream-integration.mdx:25]:
```suggestion
  In some cases, the agent will directly wrap base framework code that's invoked within your application and won't display within your IDE.
```

It's not clear what won't display within my IDE - is it "the agent won't display" or "the framework code won't display" within your IDE?

File: src/content/docs/apm/agents/nodejs-agent/installation-configuration/nodejs-agent-configuration.mdx [hunk 5]
```diff
@@ -3893,7 +3893,7 @@ Additionally, distributed tracing must be [enabled](/docs/apm/agents/nodejs-agen
 
 The application logging configuration settings require [Node.JS agent version 8.11.0 or higher](/docs/apm/agents/nodejs-agent/installation-configuration/update-nodejs-agent/).
 
-For some tips on configuring logs for the Node.js agent, see [Configure Node.js logs in context](/docs/logs/logs-context/configure-logs-context-nodejs). 
+For some tips on configuring logs for the Node.js agent, see [Configure Node.js logs in context](/docs/logs/logs-context/configure-logs-context-nodejs).
```
Comment (a-sassman) [inline src/content/docs/apm/agents/nodejs-agent/installation-configuration/nodejs-agent-configuration.mdx:3896]:
```suggestion
For tips on configuring logs for the Node.js agent, see [Configure Node.js logs in context](/docs/logs/logs-context/configure-logs-context-nodejs).
```

File: src/content/docs/apm/agents/nodejs-agent/installation-configuration/nodejs-agent-configuration.mdx [hunk 6]
```diff
@@ -4108,6 +4108,54 @@ For some tips on configuring logs for the Node.js agent, see [Configure Node.js
   </Collapser>
 </CollapserGroup>
 
+## Code level metrics [#code-level-metrics]
+
+The code level metrics configuration settings require [Node.JS agent version 9.7.5 or higher](/docs/apm/agents/nodejs-agent/installation-configuration/update-nodejs-agent/).
```
Comment (a-sassman) [inline src/content/docs/apm/agents/nodejs-agent/installation-configuration/nodejs-agent-configuration.mdx:4113]: Would it be accurate to hyphenate "code-level metrics"?

File: src/content/docs/apm/agents/nodejs-agent/installation-configuration/nodejs-agent-configuration.mdx [hunk 7]
```diff
@@ -4108,6 +4108,54 @@ For some tips on configuring logs for the Node.js agent, see [Configure Node.js
...
+
+For more details, see our documentation for [New Relic CodeStream integration](/docs/apm/agents/nodejs-agent/installation-configuration/codestream-integration).
```
Comment (a-sassman) [inline src/content/docs/apm/agents/nodejs-agent/installation-configuration/nodejs-agent-configuration.mdx:4115]:
```suggestion
For more details, see our documentation for the [New Relic CodeStream integration](/docs/apm/agents/nodejs-agent/installation-configuration/codestream-integration).
```

**Review by ghost — DISMISSED** [review]
I left some suggestions, but otherwise looks good!

File: src/content/docs/apm/agents/nodejs-agent/installation-configuration/codestream-integration.mdx (hunk 2 above)
Comment (bizob2828) [reply src/content/docs/apm/agents/nodejs-agent/installation-configuration/codestream-integration.mdx:14]: frankly this is copy/pasted from an already approved doc for the .net agent https://docs.newrelic.com/docs/apm/agents/net-agent/other-features/net-codestream-integration/

File: src/content/docs/apm/agents/nodejs-agent/installation-configuration/codestream-integration.mdx (hunk 4 above)
Comment (bizob2828) [reply src/content/docs/apm/agents/nodejs-agent/installation-configuration/codestream-integration.mdx:25]: base framework code. But this was just copied from the same doc i referenced above https://docs.newrelic.com/docs/apm/agents/net-agent/other-features/net-codestream-integration/

File: src/content/docs/apm/agents/nodejs-agent/installation-configuration/nodejs-agent-configuration.mdx (hunk 6 above)
Comment (bizob2828) [reply src/content/docs/apm/agents/nodejs-agent/installation-configuration/nodejs-agent-configuration.mdx:4113]: I'm not sure

**Review by bizob2828 — COMMENTED** [review]
Thanks for the review @a-sassman. My only concern is that most of this copy already exists on our docs website.  You may want to file a PR to fix the other language agents because I think we copied this around and Node.js was the last language to add support for this

File: src/content/docs/apm/agents/nodejs-agent/installation-configuration/nodejs-agent-configuration.mdx (hunk 6 above)
Comment (bizob2828) [reply src/content/docs/apm/agents/nodejs-agent/installation-configuration/nodejs-agent-configuration.mdx:4113]: it seems like the language is mixed from searching this repo for both `code level metrics` and `code-level metrics`

**Pull request merged**